torch_xla/core/dynamo_bridge.py

```diff
@@ -28,6 +28,25 @@
 ptxla_debug = int(os.environ.get('PT_XLA_DEBUG', '0')) == 1
 
 
+class AliasWithBufferDonorContext(object):
+
+  def __init__(self, should_alias: bool):
+    self.should_alias = should_alias
```

Comment:
Question: When do we want to set it to false? Seems like we always want to alias buffer if possible

Reply:
yea, I guess one way to do it is to let this context manager only set it to true..

Reply:
Let me adjust this along with @will-cromar 's comment in https://github.com/pytorch/xla/pull/6616